docs/how-to-contribute/general-onboarding.md

```diff
@@ -0,0 +1,65 @@
+---
+sidebar_position: 3
+title: "General Onboarding Checklist"
+sidebar_label: "General Onboarding Checklist"
+---
+# Step 1  General Setup
+- [ ] Sign up for [Discord](https://discord.gg/RFufzsqRmJ) and introduce yourself to the community
+- [ ] Sign up for [openbeta.io](https://openbeta.io/)
+- [ ] Subscribe to [our newsletter](https://docs.openbeta.io/stay-connected)
+- [ ] Connect with us on [Instagram](https://www.instagram.com/openbetaproject/)
+
+# Step 2 Deep Dive About Open Beta
+
+- [ ] Read [our roadmap](https://docs.openbeta.io/introduction/roadmap)
+- [ ] Read [our blog]()
```

Comment:
https://openbeta.substack.com/

I saw you said you were aware that the links aren't filled, but just thought I'd leave this here in case you were looking for it